keyserver/server.go

```diff
@@ -81,6 +82,8 @@ type ServerPolicies struct {
 // a mechanism to update the underlying ConiksDirectory automatically
 // at regular time intervals.
 type ConiksServer struct {
+	logger zap.Logger
```

Comment:
Ok cool! I figured you had something in mind, but I wanted to start moving things along again.